HISTORY.md

```diff
@@ -1,5 +1,19 @@
 # Release History
 
+## 0.11.2 (2020-03-17)
+
+- Fixes a bug in testing tools
```

Comment:
I know we never did this, but now or in the future it might be good to reference the bug with its GitHub id, and also spend more words to describe the actual issue.

Reply:
I mean, if in this case it is a very minor bug, then it is fine (although in that case probably we should not mention it)

Reply:
I agree. We will adopt this approach from v1 onwards. You're welcome to do it for historic releases.